Editor/MobiledgeXEditorWindow.cs

```diff
@@ -68,6 +68,35 @@ private Texture2D MakeTex(int width, int height, Color col)
         }
 
         private string sdkVersion;
+        private int selectedRegionIndex = 0;
+        private string[] regionOptions = new string[5] { "Nearest", "EU", "JP", "KR", "US" };
+        private string region;
+        public string Region
+        {
+            set { region = value; }
+            get
+            {
+                switch (region)
+                {
+                    case "EU":
+                        return EU_DME;
+                    case "KR":
+                        return KR_DME;
+                    case "JP":
+                        return JP_DME;
+                    case "US":
+                        return US_DME;
+                    case "Nearest":
+                    default:
+                        return WIFI_DME;
+                }
+            }
+        }
+        const string WIFI_DME = "wifi.dme.mobiledgex.net";
```

Comment:
Dup: public const string wifiOnlyDmeHost = wifiCarrier + "." + baseDmeHost; // but if it's not in the name space, then not terrible to duplicate among the ones below.